tests/page-objects/profile-page.ts

```diff
@@ -0,0 +1,89 @@
+import { type Page, type Locator, expect } from '@playwright/test';
+
+import { SearchPage } from './search-page';
+import { CollectionFacets } from './collection-facets';
+import { InfiniteScroller } from './infinite-scroller';
+import { SortBar } from './sort-bar';
+
+export class ProfilePage {
+  readonly page: Page;
+
+  readonly pageSummary: Locator;
+  readonly pageTabs: Locator;
+
+  readonly collectionFacets: CollectionFacets;
+  readonly infiniteScroller: InfiniteScroller;
+  readonly searchPage: SearchPage;
+  readonly sortBar: SortBar;
+
+  public constructor(page: Page) {
+    this.page = page;
+
+    this.pageSummary = page.locator('#user-summary-container');
+    this.pageTabs = page.locator(
+      '#page-container > tab-manager > div.tab-manager-container > nav.tabs-row > ul',
+    );
+
+    this.collectionFacets = new CollectionFacets(this.page);
+    this.infiniteScroller = new InfiniteScroller(this.page);
+    this.searchPage = new SearchPage(this.page);
+    this.sortBar = new SortBar(this.page);
```

Comment:
I don't think if these variables being used below?

Reply:
True, I can remove these for now. We'll most likely want to add them back as we grow the profile page tests though.

Reply:
On a second pass, some of these are now being used in new tests that were merged into this branch, and in incoming https://github.com/internetarchive/archiveorg-e2e-playwright/pull/31 too. I think it makes sense to keep these since they will imminently be useful for other profile tests being actively added.